.travis.yml

```diff
@@ -39,21 +37,20 @@ jobs:
   include:
   - stage: test
     script:
-    - jabba use "adopt@1.8-0"
     - git branch -f "$TRAVIS_BRANCH" && git checkout "$TRAVIS_BRANCH"
     - sbt -jvm-opts .jvmopts-travis ";scalafmtCheckAll;+test:compile;docs/paradox"
     name: "Compile on all Scala versions, check code style (fixed with `sbt Test/compile`)"
 
-  - stage: test
-    env: CMD="+test"
-    name: "Run tests on AdoptOpenJDK 8 (all Scala versions)"
-
   - env:
-    - JDK="adopt@1.11-0"
+    - ADOPTOPENJDK=8
```

Comment:
so this is built into travis? is this an experiment to see if it works better than Jabba or do we already know that this is what we shall use everywhere?

Reply:
It is imported from https://github.com/scala/scala-dev/blob/scala-dev/travis/default.yml (see also https://twitter.com/SethTisue/status/1239988439199580160).

I introduced it in https://github.com/akka/akka-http/pull/3137 because jabba was lagging in supported jdk version, but I don't have actual experience with it myself yet

Reply:
ok, the goal should be that we use same everywhere so we don't have to trouble shoot several things, but maybe let this mature in akka-http and here before rolling it out in all places

Reply:
We've seen our use of jabba failing quite often recently. But maybe I should remove it again so that we see how sdkman behaves in the more active Akka HTTP repository instead.

Reply:
Kept jabba for now.